.github/workflows/docker/buildwheel.sh

```diff
@@ -13,9 +13,12 @@ git config --global --add safe.directory /project
 # Fetch the full history as we'll be missing tags otherwise.
 git fetch --unshallow
```

Comment:
tsdate is Python only right, so can we ditch the docker stuff entirely?

Reply:
Yep, removed.